components/Carousel.js

```diff
@@ -21,47 +21,44 @@ class Carousel {
   }
 
   _rotate(direction) {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
-    carouselTrack.style.transition = "transform 0.6s ease-in-out";
+    this._carouselTrack.style.transition = "transform 0.6s ease-in-out";
     this._counter += direction;
-    carouselTrack.style.transform = 'translateX(' + ( -this._sizeOfSlide * this._numberOfSlidesPerPage * this._counter ) + 'px)';
+    this._carouselTrack.style.transform = 'translateX(' + ( -this._sizeOfSlide * this._numberOfSlidesPerPage * this._counter ) + 'px)';
```

Comment:
Would this look cleaner if it was made with `` instead of ''?

Reply:
What do you mean?

Reply:
Normally when you're placing variables inside of strings you use backticks. And place your variables inside with ${}